kolena/_experimental/quality_standard.py

```diff
@@ -91,3 +98,58 @@ def download_quality_standard_result(
         for model in models:
             result_dfs.append(_download_quality_standard_result(dataset, [model], metric_groups, intersect_results))
         return pd.concat(result_dfs, axis=1)
+
+
+@dataclass(frozen=True)
+class CopyQualityStandardRequest:
+    dataset_id: int
+    source_dataset_id: int
+    include_metric_groups: bool = True
+    include_test_cases: bool = True
+
+
+@with_event(event_name=EventAPI.Event.COPY_QUALITY_STANDARD_FROM_DATASET)
+def copy_quality_standards_from_dataset(
+    dataset: str,
+    source_dataset: str,
+    include_metric_groups: bool = True,
+    include_test_cases: bool = True,
+) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any]]]:
+    """
+    Create quality standards on a dataset by copying from a source dataset. Note that this operation will overwrite the
```

Comment:
`Create quality standards on a dataset` should this be `quality standard` (singular). That is how it is documented above.